storage/object/reference-content/optimize-object-storage-performance.mdx

```diff
@@ -0,0 +1,107 @@
+
+These guidelines aim to provide recommendations and methods to help you get the most out of your Object Storage.
+
+## Hardware recommendations
+
+### Network
+
+To avoid network-related bottlenecks in your infrastructure, allow as much bandwidth as possible to your Object Storage deployment.
+
+Adapt the throughput and latency of your bandwidth according to your most critical use cases:
+
+- Favor throughput over latency for concurrent requests on large multipart requests to maximize efficiency. A smaller number of objects means less per-request connection setups and Secure Socket Layer (SSL) handshakes, which generate latency for each part, but have little-to-no impact on throughput thanks to parallelization.
+
+- Favor latency over throughput for smaller-size objects (less than 1 MB), such as logs or website assets. Avoid per-request connection to remove TCP slow starts and SSL handshakes on each request. A lower latency will also reduce your Time To First Byte (TTFB) for larger objects. The tradeoff is that `ListObject` operations will take longer to complete as they will return a greater number of objects. 
+
+<Message type="tip">
+  [Prefixes](/storage/object/concepts/#prefix) and the `MaxKeys` parameter allow you to target smaller object batches.
+</Message>
+
+### CPU
+
+The CPU plays a major role in your Object Storage infrastructure, as it will determine your concurrency capacity. A good order of magnitude is 16 cores per socket to handle concurrent requests efficiently.
+
+Try various operations with a single request at a time and measure your CPU usage. The operations that have the highest CPU usage are likely to throttle your machine and create bottlenecks. These operations can be used to determine the concurrency capacity of your Object Storage deployment.
+
+For example, if the most CPU-intensive operation uses 20% of your CPU, you can execute up to five requests concurrently, to reach 100% of your processing capacity.
+
+<Message type="tip">
+  Several tools, such as [htop](https://htop.dev/) allow you to monitor your CPU activity in real time to evaluate the strain of your Object Storage workloads.
+</Message>
+
+### Geographic location
+
+The physical distance to the hardware hosting your Object Storage can also have an impact on performance, especially on latency. Make sure to benchmark the different [regions](/storage/object/concepts/##region-and-availability-zone) where Object Storage is available to compare latency on your mission-critical S3 operations.
+
+For instance, media and content distribution are often heavily affected by the physical distance between the host and the client, as objects are usually large in this scenario.
+
+<Message type="tip">
+To optimize latency and throughput on `GET` operations, Scaleway’s [Edge Services](/storage/object/how-to/get-started-edge-services/) allows for closer caching of your objects.
```

Comment:
This is currently in Private Beta by invitation only - the link to the doc has only been given to beta testers. I think it would be better to either not mention it, or remove the link to the doc and say something along the lines of "This is currently available to private beta testers only - [get in touch](https://www.scaleway.com/en/betas/#edge-services) if you're interested in joining the beta"